src/dev/build/tasks/create_archives_sources_task.ts

```diff
@@ -54,7 +54,11 @@ export const CreateArchivesSources: Task = {
             resolve(REPO_ROOT, 'config'),
             build.resolvePathForPlatform(platform, 'config'),
             {
-              select: ['serverless.yml', 'serverless.{es,oblt,security}.yml'],
+              select: [
+                'serverless.yml',
+                'serverless.{es,oblt,security}.yml',
```

Comment:
While you're here:
```suggestion
                'serverless.{chat,es,oblt,security}.yml',
```

Reply:
done

Reply:
Thanks. I see CI failed, I guess you need to merge the latest main to pick up the upstream fix.